apps/site/src/components/PageLanding/Partners.tsx

```diff
@@ -0,0 +1,43 @@
+import useTranslation from 'next-translate/useTranslation';
+import { ReactComponent as NBAIcon } from './images/partners/icon-nba.svg';
+import { ReactComponent as DrSeussIcon } from './images/partners/icon-drseuss.svg';
+import { ReactComponent as UFCIcon } from './images/partners/icon-ufc.svg';
+import { ReactComponent as SamsungIcon } from './images/partners/icon-samsung.svg';
+import { ReactComponent as UbisoftIcon } from './images/partners/icon-ubisoft.svg';
+import { ReactComponent as NFLIcon } from './images/partners/icon-nfl.svg';
+import { ReactComponent as TopshotIcon } from './images/partners/icon-topshot.svg';
+import { ReactComponent as StarIcon } from './images/partners/icon-star-green.svg';
+
```

Comment:
The brand logos take too long to appear, especially on larger screens (they should loop endlessly and not disappear after a while). Make the animation slower as well.